bin/build-scripts/bundle_dependencies

```diff
@@ -34,7 +34,9 @@ build() {
   echo -n "Building $(basename $dest): deps... "
   browserify --list -e "$@" | sed "s!$(pwd)/!!g" | grep -v build-time > "$(depfile)"
   echo -n "bundle... "
-  browserify -o "$dest" \
+  NODE_ENV=production browserify -o "$dest" \
+    -g [ envify --NODE_ENV production ] \
+    -g uglifyify \
```

Comment:
What does this mean in relation to the `$NO_UGLIFY` block below? Even if we specify NO_UGLIFY, the bundle would still be minified, no?

Reply:
Yup, see #3433 for the rest of the story